packages/core/utils/src/replaceBundleReferences.js

```diff
@@ -32,117 +32,110 @@ export async function replaceBundleReferences({
   bundleGraph,
   contents,
   map,
-  getInlineReplacement,
-  getInlineBundleContents,
+  replaceInline,
+  replaceUrls = true,
+  relative = true,
 }: {|
   bundle: Bundle,
   bundleGraph: BundleGraph,
   contents: string,
-  getInlineReplacement: (
-    Dependency,
-    ?'string',
-    string,
-  ) => {|from: string, to: string|},
-  getInlineBundleContents: (
-    Bundle,
-    BundleGraph,
-  ) => Async<{|contents: Blob, map: ?(Readable | string)|}>,
+  relative?: boolean,
+  replaceInline?: {|
```

Comment:
Why are these two methods grouped into an object?

Reply:
One was essentially a subset of the other. I also added the ability to only replace inline content. Replacing urls and replacing inline content should probably be split into two methods and allow callers to compose them, but I made one method with multiple switches to avoid iterating over every dependency in the bundle multiple times.

I'll explore separating them (and therefore iterating every dependency twice in the e.g. css case), as I imagine the performance impact isn't as significant as I thought.